src/main/scala/cromwell/binding/Call.scala

```diff
@@ -31,13 +31,16 @@ case class Call(alias: Option[String], taskFqn: FullyQualifiedName, task: Task,
     else throw new UnsupportedOperationException("parent is write-once")
   }
 
+  private def unsatisfiedTaskInputs: Seq[TaskInput] = task.inputs.filterNot {case i => inputMappings.contains(i.name)}
   /**
    * Map of task-local input names to type for every input that this
```

Comment:
I'm not sure what this comment is saying, but I'm pretty sure it's not what I thought this function was doing. Can you rephrase the purpose of this function in long form here in the github commentary?